library/Icinga/Web/Controller/ActionController.php

```diff
@@ -428,6 +435,13 @@ protected function redirectXhr($url)
         $response->redirectAndExit($url);
     }
 
+    /**
+     * Redirect http requests to the specified url
```

Comment:
*Issue a redirect that's performed as a native HTTP request by the client*

*This will effectively reload the window*